spiped/main.c

```diff
@@ -225,15 +253,16 @@ main(int argc, char * argv[])
 		}
 	}
 
+	if (signal(SIGTERM, graceful_shutdown_handler) == SIG_ERR) {
+		warnp("Failed to bind SIGTERM signal handler");
```

Comment:
If this fails, we should exit.  This is different from when we bind signals in order to work around the docker bug; in that case we specifically *do not* want to exit since that's a workaround.  (Rule #1 of workarounds: Don't break anything else.)